core/src/navigation/ProductSwitcher.html

```diff
@@ -0,0 +1,83 @@
+<div class="fd-shellbar__action fd-shellbar__action--collapsible">
+  <div class="fd-product-switcher">
+    <div class="fd-popover fd-popover--right">
+      <div class="fd-popover__control" on:click="event.stopPropagation()">
+        <button
+          class="fd-button--shell sap-icon--grid"
+          aria-expanded="{dropDownStates.productSwitcherPopover || false}"
+          aria-haspopup="true"
+          on:click="toggleDropdownState('productSwitcherPopover')"
+        ></button>
+        <div
+          class="fd-popover__body fd-popover__body--right"
+          aria-hidden="{!(dropDownStates.productSwitcherPopover || false)}"
+          id="productSwitcherPopover"
+        >
+          <div class="fd-product-switcher__body">
+            <nav>
+              <ul>
+                {#if productSwitcherItems && productSwitcherItems.length} {#each
```

Comment:
This `if` probably should be on the top of this html file, since when the product switcher has not items I see the following:
![screen shot 2019-01-23 at 11 18 47](https://user-images.githubusercontent.com/8238188/51599877-f6a54c80-1f00-11e9-9466-5eadd69729d0.png)

Reply:
done